test/test_retrieve_utils.py

```diff
@@ -1,242 +0,0 @@
-        pdf_file_path = os.path.join(test_dir, "example.pdf")
-        assert "".join(expected_text.split()) == "".join(extract_text_from_pdf(pdf_file_path).strip().split())
-
-    def test_split_files_to_chunks(self):
-        pdf_file_path = os.path.join(test_dir, "example.pdf")
-        txt_file_path = os.path.join(test_dir, "example.txt")
-        chunks = split_files_to_chunks([pdf_file_path, txt_file_path])
-        assert all(
-            isinstance(chunk, str) and "AutoGen is an advanced tool designed to assist developers" in chunk.strip()
-            for chunk in chunks
-        )
-
-    def test_get_files_from_dir(self):
-        files = get_files_from_dir(test_dir, recursive=False)
-        assert all(os.path.isfile(file) for file in files)
-        pdf_file_path = os.path.join(test_dir, "example.pdf")
-        txt_file_path = os.path.join(test_dir, "example.txt")
-        files = get_files_from_dir([pdf_file_path, txt_file_path])
-        assert all(os.path.isfile(file) for file in files)
-        files = get_files_from_dir(
-            [
-                pdf_file_path,
-                txt_file_path,
-                os.path.join(test_dir, "..", "..", "website/docs"),
-                "https://raw.githubusercontent.com/microsoft/autogen/main/README.md",
-            ],
-            recursive=True,
-        )
-        assert all(os.path.isfile(file) for file in files)
-        files = get_files_from_dir(
-            [
-                pdf_file_path,
-                txt_file_path,
-                os.path.join(test_dir, "..", "..", "website/docs"),
-                "https://raw.githubusercontent.com/microsoft/autogen/main/README.md",
-            ],
-            recursive=True,
-            types=["pdf", "txt"],
-        )
-        assert all(os.path.isfile(file) for file in files)
-        assert len(files) == 3
-
-    def test_is_url(self):
-        assert is_url("https://www.example.com")
-        assert not is_url("not_a_url")
-
-    def test_create_vector_db_from_dir(self):
-        db_path = "/tmp/test_retrieve_utils_chromadb.db"
-        if os.path.exists(db_path):
-            client = chromadb.PersistentClient(path=db_path)
-        else:
-            client = chromadb.PersistentClient(path=db_path)
-            create_vector_db_from_dir(test_dir, client=client)
-
-        assert client.get_collection("all-my-documents")
-
-    def test_query_vector_db(self):
-        db_path = "/tmp/test_retrieve_utils_chromadb.db"
-        if os.path.exists(db_path):
-            client = chromadb.PersistentClient(path=db_path)
-        else:  # If the database does not exist, create it first
-            client = chromadb.PersistentClient(path=db_path)
-            create_vector_db_from_dir(test_dir, client=client)
-
-        results = query_vector_db(["autogen"], client=client)
-        assert isinstance(results, dict) and any("autogen" in res[0].lower() for res in results.get("documents", []))
-
-    def test_custom_vector_db(self):
```

Comment:
This is still needed since we need to keep the extensibility of adding new dbs in users' code.

Reply:
reverted